solr/core/src/java/org/apache/solr/cloud/api/collections/SplitShardCmd.java

```diff
@@ -163,6 +192,10 @@ public boolean split(ClusterState clusterState, ZkNodeProps message, NamedList<O
 
     RTimerTree t;
     if (ccc.getCoreContainer().getNodeConfig().getMetricsConfig().isEnabled()) {
+      // 1. verify that there is enough space on disk to create sub-shards
+      log.info(
```

Comment:
Sometimes we don't check disk space, but we'd log and calculate timings.  Can you move the check that is currently inside checkDiskSpace() (which @heythm added) to here so that we do neither if we don't actually check disk space?